packages/neuron-wallet/src/models/chain/live-cell.ts

```diff
@@ -64,8 +64,8 @@ export default class LiveCell {
       : null
 
     return new LiveCell(
-      cell.outPoint.txHash,
-      cell.outPoint.index,
+      cell.outPoint!.txHash,
+      cell.outPoint!.index,
```

Comment:
The `Cell` in Lumos does not distinguish between whether it is an input or output cell. Metaphorically, a `LiveCell` implies that a cell must be an input cell. Therefore, we should throw an error here if a cell does not have an `OutPoint`, meaning it is not an input cell

```js

asserts(cell.outPoint, `Cell must be a live cell`)

return new LiveCell(
  cell.outPoint.txHash,
  cell.outPoint.index,
)

```

Reply:
@homura  Is this OK ? 

Since the original code only type checks `cell.outPoint`, so I only replace `Cell` with `CellWithOutPoint`.